gengokit/httptransport/templates/server.go

```diff
@@ -188,7 +197,7 @@ func errorEncoder(_ context.Context, err error, w http.ResponseWriter) {
 }
 
 type errorWrapper struct {
-	Error string ` + "`json:\"error\"`\n" + `
+	Error string ` + "`" + `json:"error"` + "`" + `
```

Comment:
This is a raw string literal. Using the back quote or grave symbol `. We do this so we do not have to escape quotes. Though the only way to have a back quote in a raw string literal is to concatenate it with a string using quotes.

> [Raw string literals are character sequences between back quotes, as in `. Within the quotes, any character may appear except back quote.](https://golang.org/ref/spec#String_literals)